js/apps/events-page/controllers/eventRegistrationCtrl.js

```diff
@@ -0,0 +1,50 @@
+((app) => {
+	'use strict';
+
+	const EventRegistrationCtrl = function EventsPageCtrl($window, $scope, $routeParams, eventsService, registrationService, dateUtilityService) {
+		const id = $routeParams.id;
+
+		const vm = this;
+
+		vm.isGroup = 'false';
+		vm.isSubmitted = false;
+		vm.isLoadingResults = false;
+
+		vm.submitHandler = () => {
+			vm.isLoadingResults = true;
+
+			const postModel = {
```

Comment:
if there isnt going to be any other postModels remove one of the outer object

Reply:
That's how the vendor API wants it, wrapped in another object. However, you have a point. I just have to make some small changes to the web service to match.

Reply:
10-4